_docs/index.md

```diff
@@ -12,6 +12,8 @@ type: markdown
 
 # Welcome
 
+Here you will find the latest ({{site.data.istio.version}}) documentation, follow this link for the [documentation for the latest stable release (0.1)](https://istio.io/v-0.1/docs/)
```

Comment:
This shouldn't be the first thing at the top of the Welcome page. How about this:

- Change the Welcome header to be Welcome to Istio <version number>

- Add a new paragraph at the end of the page that points to the stable version